### PR TITLE
[Review Again] Updates the cloning manual

### PR DESCRIPTION
**Pull request by ghost**
:cl: Mark9013100
tweak: The Medical Cloning manual has been updated.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.) 

Because the book's old as fuck and suffered from corrupted lettering. Removes some excess comments as well.

### CODE REVIEW

**Comment by ACCount12** [comment]
With BYOND being BYOND, we really should not use non-ascii characters unless absolutely necessary.

**Comment by Mark9013100** [comment]
forgot to add something, DNM please

File: code/game/objects/items/manuals.dm [hunk 1]
```diff
@@ -150,56 +148,52 @@
...
-					<li><a href='#7'>Put body in morgue</a></li>
-					<li><a href='#8'>Await cloned body</a></li>
-					<li><a href='#9'>Use the clean SW injector</a></li>
-					<li><a href='#10'>Give person clothes back</a></li>
-					<li><a href='#11'>Send person on their way</a></li>
+					<li><a href='#1'>Acquire body/head/brain</a></li>
+					<li><a href='#2'>Put body/head/brain in cloning machine</a></li>
+					<li><a href='#3'>Scan body/head/brain</a></li>
+					<li><a href='#4'>Clone body/head/brain</a></li>
+					<li><a href='#5'>Get Mannitol, mutadone, or a clean SE for the clone</a></li>
```
Comment (Dorsisdwarf) [inline code/game/objects/items/manuals.dm]: Mannitor and mutadone/ SE, this sentence says "you need 1 of 3" when actually you need 2

Comment (Mark9013100) [reply code/game/objects/items/manuals.dm]: I'll fix that in a sec

**Comment by Mark9013100** [comment]
messed up somewhere, going to need to give this one last edit

**Comment by Mark9013100** [comment]
Alright, should be ready once travis finishes

File: code/game/objects/items/manuals.dm [hunk 2]
```diff
@@ -9,8 +9,10 @@
 /obj/item/book/manual/engineering_particle_accelerator
 	name = "Particle Accelerator User's Guide"
 	icon_state ="bookParticleAccelerator"
-	author = "Engineering Encyclopedia"		 // Who wrote the thing, can be changed by pen or PC. It is not automatically assigned
+	author = "Engineering Encyclopedia"		  // Whoever wrote the paper or book, can be changed by pen or PC. It is not automatically assigned.
```
Comment (ninjanomnom) [inline code/game/objects/items/manuals.dm]: >changing incorrect grammar in a comment to even more incorrect grammar

**Pull request merged**